heudiconv/dicoms.py

```diff
@@ -531,19 +532,12 @@ def get_datetime_from_dcm(dcm_data: dcm.FileDataset) -> Optional[datetime.dateti
     3. SeriesDate & SeriesTime  (0008,0021); (0008,0031)
 
     """
-    acq_date = dcm_data.get("AcquisitionDate", "").strip()
-    acq_time = dcm_data.get("AcquisitionTime", "").strip()
-    if acq_date and acq_time:
-        return strptime_micr(acq_date + acq_time, "%Y%m%d%H%M%S[.%f]")
-
-    acq_dt = dcm_data.get("AcquisitionDateTime", "").strip()
-    if acq_dt:
-        return strptime_micr(acq_dt, "%Y%m%d%H%M%S[.%f]")
-
-    series_date = dcm_data.get("SeriesDate", "").strip()
-    series_time = dcm_data.get("SeriesTime", "").strip()
-    if series_date and series_time:
-        return strptime_micr(series_date + series_time, "%Y%m%d%H%M%S[.%f]")
+    if "AcquisitionDate" in dcm_data and "AcquisitionTime" in dcm_data:
+        return strptime_dcm_da_tm(dcm_data, "AcquisitionDate", "AcquisitionTime")
```

Comment:
FWIW this is not functionally equivalent to above since we did `strip()` on values, so if string of spaces is a value for some reason (don't want to dig now why we even did such safe guarding), it would not be considered, which is the case here.  Could you adjust so we stay 100% consistent with prior implementation?

Reply:
not sure why it was marked resolved since  code remains the same -- if empty value we would proceed to the function and return